packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1272,7 +1305,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
    * because we want to keep the event handling interface
    */
   setStateAndInvokeHandler = (
-    handlerName: keyof DropdownProps,
+    handlerNames: string[],
```

Comment:
Why did we change `keyof DropdownProps` to `string`? Can't we make this array of keyof DropdownProps?